doc/spec.md

```diff
@@ -834,6 +842,10 @@ applied to sets.  The right operand of the `|` operator may be any
 iterable value.  The binary `in` operator performs a set membership
 test when its right operand is a set.
 
+The binary `^` operator performs symmetric difference of two sets.
+The resulting set will have an elements of either first or second
```

Comment:
typo: "an elements"

But I think you can drop the second sentence (The resulting...) as this section is just an overview of the available operators, not their semantics in detail.